### PR TITLE
Adding pants and pockets

### PR DESCRIPTION
**Pull request by ob1jedi**
Please review this, @casey-collab

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2035)

### CODE REVIEW

File: english/look-in-pants-pocket/inleftpocket.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+You find a folded up piece of paper
```
Comment (casey-collab) [inline english/look-in-pants-pocket/inleftpocket.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: english/look-in-pants-pocket/inrightpocket.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+You find a green marshmallow
```
Comment (casey-collab) [inline english/look-in-pants-pocket/inrightpocket.md]: This file has also been added to a directory that already contained a page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
@casey-collab, please take another look

File: english/look-in-pants-pocket/right-pocket/inrightpocket.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+You find a green marshmallow
+
+[Done here](../marshmallow.md)
```
Comment (casey-collab) [inline english/look-in-pants-pocket/right-pocket/inrightpocket.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/look-in-pants-pocket/search-pockets.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+[Check the left pants pocket](./left-pocket/inleftpocket.md)
+
+[Check the left pants pocket](./right-pocket/inleftpocket.md)
```
Comment (casey-collab) [inline english/look-in-pants-pocket/search-pockets.md]: This link is also invalid.

File: english/look-in-pants-pocket/right-pocket/inrightpocket.md [hunk 5]
```diff
@@ -0,0 +1,3 @@
+You find a green marshmallow
```
Comment (casey-collab) [inline english/look-in-pants-pocket/right-pocket/inrightpocket.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**